src/main/resources/com/cloudbees/hudson/plugins/folder/AbstractFolder/tasks-new_ja.properties

```diff
@@ -20,4 +20,5 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-delete={0}\u306e\u524a\u9664
+delete={0}\u306E\u524A\u9664
+delete.confirm={0} "{1}"\u3092\u524A\u9664\u3057\u3066\u3082\u3088\u308D\u3057\u3044\u3067\u3059\u304B?
```

Comment:
I hope that the translation is correct. I copied this from the configurable.jelly from core

Reply:
As a rule, if you are not a native speaker, you should not add or edit entries in translations. If a given key is deleted from the base (English) file, or changes meaning or format, delete it from translations. Let translators keep up later if they want.

(https://github.com/jenkinsci/jep/blob/master/jep/216/README.adoc would be clearer I think.)

Reply:
ok removed it